### PR TITLE
3dVolViewer adds

### PR DESCRIPTION
**Pull request by kerautret**
# PR Description
Improvement with the possibility to read input image of type double (with ITK).    New possibility to selection the voxel in order to display image intensity.  
# Checklist

- [x] Doxygen documentation of the code completed (classes, methods, types, members...).
- [x] Main tool doxygen documentation (following existing documentation of [DGtalTools documentation](http://dgtal.org/doc/tools/nightly/).
- [x] Check if it follows the tools structure described in [CONTRIBUTING.md](https://github.com/DGtal-team/DGtalTools/blob/master/CONTRIBUTING.md)
- [x] New entry in the [ChangeLog.md](https://github.com/DGtal-team/DGtalTools/blob/master/ChangeLog.md) added.
- [x] Update the readme with potentially a screenshot of the tools if it applies. 
- [x] No warning raised in Debug ```cmake``` mode (otherwise, Travis C.I. will fail).

### CODE REVIEW

File: ChangeLog.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
 # DGtalTools 1.2
 - *global*
+  - Fix itk2vol and fix ITK cmake configuaration that was making wrong the ITK image read.
```
Comment (dcoeurjo) [inline ChangeLog.md:3]:
```suggestion
  - Fix itk2vol and fix ITK cmake configuaration that was making issues with the ITK image read.
```

File: ChangeLog.md [hunk 2]
```diff
@@ -1,14 +1,16 @@
...
+    (Bertrand Kerautret [#393](https://github.com/DGtal-team/DGtalTools/pull/393))
   - Travis: Fix old default osx_image with xcode12.2 and remove non used boost
     cmake references. (Bertrand Kerautret [#394](https://github.com/DGtal-team/DGtalTools/pull/394)) 
 
-- *visualisation*
+- *visualisation*:
+  - 3dVolViewer: improvement with the possibility to read input image of type double (with ITK).
```
Comment (dcoeurjo) [inline ChangeLog.md:9]:
```suggestion
  - 3dVolViewer: improvement of the possibility to read input image of type double (with ITK).
```

File: ChangeLog.md [hunk 3]
```diff
@@ -1,14 +1,16 @@
...
+    New possibility to selection the voxel in order to display image intensity.
```
Comment (dcoeurjo) [inline ChangeLog.md:10]:
```suggestion
    New possibility to select the voxel in order to display image intensity.    
```

**Comment by kerautret** [comment]
thanks @dcoeurjo for the corrections, done

**Pull request merged**